src/lang/en-US.json

```diff
@@ -68,6 +68,8 @@
   "barracks_value_1024": "Radiant Top Melee",
   "barracks_value_2048": "Radiant Top Ranged",
 
+ "building_ancient": "Ancient",
```

Comment:
can you double-check to see if there's a string for this already?

Reply:
there is a  string for the rank "Ancient" but not the building. They are translated differently for example in German.